uktrade/data-hub-frontend
# Update yarn.lock to include Babel update

## Description of change

Updating yarn.lock so the the correct version of Babel is downloaded.

## Test instructions

Run `yarn` without `node_modules` dir and check if `yarn.lock` stayed unchanged.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied. 
https://github.com/uktrade/data-hub-frontend/blob/develop/docs/Code%20review%20guidelines.md"
